### PR TITLE
Rename and move activation routines from math.h

### PR DESCRIPTION
**Pull request by emcastillo**
Final PR in the math.cc refactoring series.

Renames the routines/{math.h, math.cc} files to activation

### CODE REVIEW

**Comment by emcastillo** [comment]
There were two tests for the Sigmoid, I added both in test_activation, but maybe they should be combined?

File: chainerx_cc/chainerx/routines/activation.cc [hunk 1]
```diff
@@ -9,22 +8,16 @@
...
 #include "chainerx/array.h"
-#include "chainerx/backprop_mode.h"
-#include "chainerx/backward_builder.h"
-#include "chainerx/backward_context.h"
 #include "chainerx/dtype.h"
 #include "chainerx/enum.h"
 #include "chainerx/error.h"
 #include "chainerx/graph.h"
 #include "chainerx/macro.h"
+#include "chainerx/routines/activation.h"
```
Comment (hvy) [inline chainerx_cc/chainerx/routines/activation.cc]: Can you put this corresponding header-inclusion at the top of this file?

**Comment by hvy** [comment]
How about aligning the Python test style to the test for ReLU?

**Comment by emcastillo** [comment]
You mean to change the sigmoid tests to the same as the Relu?

**Comment by hvy** [comment]
Exactly, while also unifying the two tests. While the parameterized inputs will differ, is is possible?

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
LGTM, CI failure seems unrelated.

**Pull request merged**